### ISSUE TITLE
Update year in all licence headers

### ISSUE THREAD

**Issue by adiherzog** [scenarioo/scenarioo#431]

**Comment by xeronimus** [comment]
Is it still necessary/best practice to include the license header in every source file?

What about:
- LICENSE file on project root
- add license header to all files (that get exposed) during build.

**Comment by adiherzog** [comment]
Yeah, we could probably remove the licence everywhere and just put it in one LICENCE file. This should be ok from a legal perspective.

@bruderol What do you think about that?

**Comment by bruderol** [comment]
We looked at this, and it seems (from a strictly legal perspective) that it is necessary to put the license header in all source file headers. That's why we did so. I not remember anymore, who told us so or where we read about that. I think, that some licences even mention this, that such a header is necessary in all source files, if I remember it correctly.

**Comment by georgseeli** [comment]
You are using GPL. FSF recommends to add license information to every source file. Also the GPL guides you to add some text to all file headers together with the copyright info.
See https://www.gnu.org/licenses/gpl-howto.de.html
One more thing: on Github, the license file should be called LICENSE or LICENSE.md. This shows a nice, concise information about the license when a user selects the license file on Github (see scenarioo-js).
